problem2.swift

```diff
@@ -2,7 +2,17 @@
 import Foundation
 
 let y = Double(readLine()!)!
-
+var x = y - 0.5
+var i = 0
+// Repeat bablyonian method 13 times
```

Comment:
Why 13? Where'd you get this number? You should explain by putting 13 in a descriptively named constant and commenting why you choose 13. Like we've said before, don't use magic numbers!